### ISSUE TITLE
empty-line-between-blocks doesn't allow comments

### ISSUE THREAD

**Issue by bgriffith** [sasstools/sass-lint#12]
First off - great work! Super excited to adopt this so have had a play around!

Currently the `empty-line-between-blocks` rule doesn't allow a comment to precede a block. Is this intentional? If so, can we have a config parameter to allow this? Currently the following returns an error/warning.

```scss


// This is a comment
.block {

}


```

Looking forward to the release!

**Comment by Snugug** [comment]
Can you provide an expanded example where this is not working?

**Comment by bgriffith** [comment]
Apologies, it only appears to happen with nested blocks.

```scss

// This comment doesn't cause an error/warning
.block {

  // nested block - comment causes an error/warning
  .nested-block {
    color: orange;
  }

  // BEM element - comment causes an error/warning
  &__element {
    color: pink;
  }

  // BEM modifier - comment causes an error/warning
  &--modifier {
    color: purple;
  }

  // pseudo element - comment causes an error/warning
  &:before {
    color: blue;
  }

  // pseudo element - comment causes an error/warning
  &::first-letter {
    color: green;
  }
}
```

**Comment by Snugug** [comment]
Thanks. I'll add a test for this.
